packages/build-utils/test/unit.test.ts

```diff
@@ -538,6 +538,83 @@ it('should support experimentalStreamingLambdaPath correctly', async () => {
   );
 });
 
+it('should support chain correctly', async () => {
+  new Prerender({
+    expiration: 1,
+    fallback: null,
+    group: 1,
+    bypassToken: 'some-long-bypass-token-to-make-it-work',
+    chain: undefined,
+  });
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** split into separate tests

In this or a follow-up PR: Can you split these into separate tests? When one of these fails, the message will be "should support chain correctly", which isn't very specific.

It also makes it harder to review. I don't know what's supposed to be correct or not about these different cases.‏